### ISSUE TITLE
Checking Config

### ISSUE THREAD

**Issue by Gemzerella** [pi-node/instructions#124]
I 'believe' my node is all setup and working as it should be, the blockchain is up to date etc...

In troubleshooting, it says this: Supporting other nodes: No - I also don't have any inbound connections showing, only outbound.

I think my firewall settings and ports are open, can anyone confirm if there is something else I need to check?

Many Thanks

**Comment by ihatejam** [comment]
you're good to go! You will very rarely end up supporting other nodes -- the number of inbound connections will fluctuate between and 0 and 64 -- but is usually on the low side - zero is not uncommon, and nothing to worry about.

**Comment by Gemzerella** [comment]
Thank you so much! That's awesome :-)

Do we know any more about when people will get told if they will continue to run the nodes and complete KYC etc?

**Comment by pilgrim103** [comment]
No, they have bigger issues tonight.  Servers/connectivity down for mobile phone app.

**Comment by Gemzerella** [comment]
Thanks for all the updates guys, really appreciated!
